frontend/package.yaml

```diff
@@ -7,7 +7,13 @@ license: BSD3
 github: dorchard/granule
```

Comment:
This seems like it's out of date!

Reply:
Indeed, thank you for that, @brendanzab!